css/notes.css

```diff
@@ -73,14 +70,6 @@
     -webkit-user-select: none;
 }
 
-/* TODO: move into core app styles */
-#app-content.loading {
-    background: url('../img/loading.gif') no-repeat center;
-    /* Overrides the snap.js animation making the loading icon to fly in app-content. */
-    transition: none !important;
-    -webkit-transition: none !important;
```

Comment:
These transition rules I think need to stay because otherwise, as the comment says, the loading icon will fly into the app-content from the top left.